queries/latex/highlights.scm

```diff
@@ -0,0 +1,165 @@
+;; Math
+[
+ (displayed_equation)
+ (inline_formula)
+] @text.math
+
+;; This highlights the whole environment like vimtex does
+((environment
+  (begin
+   name: (word) @_env)) @text.math
+   (#match? @_env "^(displaymath|equation|multline|eqnarray|align)[*]?$"))
+
+;; This at the begining of the file would be the alternative to highlight
+;; only the interior of the environment
+;((environment
+  ;(begin
+   ;name: (word) @_env) @none
+   ;(end) @none) @text.math
+   ;(#match? @_env "^(displaymath|eqn|eqnarray)[*]?$"))
+
+[
+  (generic_command_name)
+  "\\newcommand"
+  "\\renewcommand"
+  "\\DeclareRobustCommand"
+  "\\DeclareMathOperator"
+  "\\newglossaryentry"
+  "\\caption"
+  "\\cite"
+  "\\label"
+  "\\newlabel"
+  "\\label"
+  "\\ref"
```

Comment:
maybe add `\cref` (or `\*ref`?)

Reply:
Only `\\cref` is a valid node.

Reply:
you mean, something like \eqref...

Reply:
Yes, exactly. (And `cleveref` defines several variations on `\cref`; then there's `autoref` etc.)

Reply:
But `eqref` and `cref` are the main things I use and would definitely miss.

Reply:
Found another one in one of my files: `\nameref`